src/main/java/core/basesyntax/services/transaction/SupplyTransaction.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.services.transaction;
+
+import core.basesyntax.dbreport.ReportStorage;
+
+public class SupplyTransaction implements Transaction {
+    @Override
+    public void doFruitTransaction(String fruit, int quantity) {

```

Comment:
```suggestion
public class SupplyOperationHandler implements OperationHandler {
    @Override
    public void handle(String fruit, int quantity) {
```
fix other handler classes